electrochemistry.ttl

```diff
@@ -3857,9 +4176,9 @@ Cl2, Hg | Hg2SO4, and Hg | HgO, can be used as reference electrodes in aqueous s
 
 ###  http://emmo.info/electrochemistry#electrochemistry_9a823d64_9d10_4a29_9cbd_9bbdad7985bc
 :electrochemistry_9a823d64_9d10_4a29_9cbd_9bbdad7985bc rdf:type owl:Class ;
-                                                       rdfs:subClassOf :electrochemistry_0f007072-a8dd-4798-b865-1bf9363be627 ,
+                                                       rdfs:subClassOf :electrochemistry_0f007072_a8dd_4798_b865_1bf9363be627 ,
                                                                        [ rdf:type owl:Restriction ;
-                                                                         owl:onProperty :electrochemistry_860aa941_5ff9_4452_8a16_7856fad07bee ;
+                                                                         owl:onProperty emmo:EMMO_dba27ca1_33c9_4443_a912_1519ce4c39ec ;
```

Comment:
I think we can't avoid having object properties for things like hasPositiveElectrode, hasNegativeElectrode, hasActiveMaterial, and hasElectrolyte, because they are so foundational to electrochemical systems. I see a two-part test for deciding "is this object property necessary?": (i) is it widely used in the field and (ii) does it add some semantic meaning to its domain/range. In this case, the answer is (i) yes and (ii) yes.